controllers/podchaos/podfailure/types.go

```diff
@@ -238,13 +258,16 @@ func (r *endpoint) recoverPod(ctx context.Context, pod *v1.Pod, podchaos *v1alph
 
 	for index := range pod.Spec.Containers {
 		name := pod.Spec.Containers[index].Name
-		_ = utils.GenAnnotationKeyForImage(podchaos, name)
+		key := utils.GenAnnotationKeyForImage(podchaos, name)
 
 		if pod.Annotations == nil {
 			pod.Annotations = make(map[string]string)
 		}
-
-		// FIXME: Check annotations and return error.
+		// Check annotations and return error
+		if _, ok := pod.Annotations[key]; !ok {
+			r.Log.Error(errNotOperatedChaos, "the pod not operated by podChaos", "namespace", pod.Namespace, "name", pod.Name)
```

Comment:
how about using `continue` and just print a warn log?

for example, container 1 is not operated, and container2 is operated, these codes will return an error(this error will be ignored later), but the pod will not delete, the container2 can not be recovered.